app/views/alchemy/admin/pages/_page.html.erb

```diff
@@ -18,17 +18,21 @@
       {{#if definition_missing}}
         <%= page_layout_missing_warning %>
       {{else}}
-        {{#unless permissions.edit}}
-          <%= page_roles_denial_warning %>
-          {{else}}
+        {{#if permissions.edit}}
```

Comment:
Since you renamed it above, shouldn't this be `if permissions.edit_content` now?